core/tests/performance_framework/perf_services.py

```diff
@@ -130,12 +134,14 @@ def get_page_metrics_from_cached_session(self, page_url):
         return perf_domain.PageSessionMetrics(
             page_session_stats=har_dict, page_session_timings=None)
 
-    def get_page_timings_for_url(self, page_url):
+    def get_page_timings_for_url(self, page_url, func_call):
         """Returns a PageSessionMetrics domain object initialized using
         page load timings for a page URL.
         """
         with Xvfb() as _:
```

Comment:
Can you please restructure this to not use a callback function? You might as well remove the ``with Xvfb() as _`` here, rather than waiting for #2184 for to be completed.